# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: D:\CM-work\hudson\plugins\description-setter
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: src/main/java/hudson/plugins/descriptionsetter/DescriptionSetterPublisher.java
--- src/main/java/hudson/plugins/descriptionsetter/DescriptionSetterPublisher.java Base (BASE)
+++ src/main/java/hudson/plugins/descriptionsetter/DescriptionSetterPublisher.java Locally Modified (Based On LOCAL)
@@ -78,9 +78,9 @@
 			}
           } else { // Hard Code
             if (setForFailed && build.getResult().isWorseThan(Result.UNSTABLE)) {
-                build.setDescription(regexpForFailed);
+                build.setDescription(build.getEnvironment(listener).expand(regexpForFailed));
             } else if (setForFailed || build.getResult().isBetterOrEqualTo(Result.UNSTABLE)) {
-                build.setDescription(regexp);
+                build.setDescription(build.getEnvironment(listener).expand(regexp));
             }
           }
 		} catch (IOException e) {
